layouts/shortcodes/version_files.md

```diff
@@ -0,0 +1,19 @@
+|#|File | Version |
+-|-----|---------|
+|1|`includes/version.php`| alpha: `define('PROJECT_VERSION_MINOR', '1.0-alpha1');`|
+|||final: `define('PROJECT_VERSION_MINOR', '1.0');`|
+|2|`zc_install/includes/version.php`|alpha: `define('PROJECT_VERSION_MINOR', '1.0-alpha1');`|
```

Comment:
Is it worth mentioning that the zc_install version.php should (pretty much always) be identical to the /includes/version.php ?

(And yes, for those who ask "why duplicate the file?", it's intentional, so that zc_install can stand alone, because it *is* version-specific, and needs to be in sync with the rest of the zc_install files.)

Reply:
added - will be updated in next push.